manifest.versions.json

```
@@ -99,10 +99,10 @@
     "chisel|8.0|ref": "$(chisel|9.0|ref)",
     "chisel|6.0|ref": "$(chisel|9.0|ref)",
 
-    "dotnet|6.0|product-version": "6.0.28",
-    "dotnet|7.0|product-version": "7.0.17",
-    "dotnet|8.0|product-version": "8.0.3",
-    "dotnet|9.0|product-version": "9.0.0-preview.3",
+    "dotnet|6.0|product-version": "6.0.29",
+    "dotnet|7.0|product-version": "7.0.18",
+    "dotnet|8.0|product-version": "8.0.4",
+    "dotnet|9.0|product-version": "9.0.0-preview.2",
```

This will require a readme update.

```suggestion
    "dotnet|9.0|product-version": "9.0.0-preview.3",
```